addons/addon-environment-sc-api/packages/environment-sc-workflow-steps/lib/plugins/workflow-steps-plugin.js

```diff
@@ -31,6 +31,12 @@ const startEC2Yaml = require('../steps/start-ec2-environment/start-ec2-environme
 const stopEC2 = require('../steps/stop-ec2-environment/stop-ec2-environment-sc.js');
 const stopEC2Yaml = require('../steps/stop-ec2-environment/stop-ec2-environment-sc.yml');
 
+const startRStudio = require('../steps/start-rstudio-environment/start-rstudio-environment-sc.js');
+const startRStudioYaml = require('../steps/start-rstudio-environment/start-rstudio-environment-sc.yml');
+
+const stopRStudio = require('../steps/stop-rstudio-environment/stop-rstudio-environment-sc.js');
```

Comment:
Is stop-rstudio-environment different from stop a normal ec2 environment?

Reply:
No, it's the same but kept it separate to avoid user confusion when searching for workflow output in the SWB "Workflows" tab. (To check workflow status, Stop RStudio workflow is more intuitive than Stop EC2 workflow)

Reply:
No, it's not. I agree with what you're getting at, it can be consolidated into common code in the future.

Reply:
Maybe we can just reuse the current codes by require the existing workflow and workflow steps if they're exactly the same?

Reply:
Sure, I'll update it.